backend/tests/database.py

```diff
@@ -0,0 +1,79 @@
+from datetime import datetime
+import unittest
+from config import db
+from database.notification import Notification
+from database.receipt import Receipt
+from database.session import Session
+
+MAGIC_TEXT = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit.'
+
+class DatabaseTest(unittest.TestCase):
```

Comment:
Good job on doing the testing ! I have some suggestions:
Can you split this database.py files into smaller files? 
Because as we go, each fields will require more testing, it will be very long if we throw all the testing into 1 file, you can create like, notification_test.py, receipt_test.py, session_test.py.

Reply:
Fixed in 2edacf3 :)